pkgs/development/tools/gomplate/default.nix

```diff
@@ -1,9 +1,16 @@
+# Gomplate 3.x does not build with go > 1.20.
+# Version 4 of gomplate (yet unreleased) should not have this issue.
+#
+# see https://github.com/hairyhenderson/gomplate/issues/1872
+
 { lib
-, buildGoModule
+#, buildGoModule
```

Comment:
```suggestion
# switch back to buildGoModule once upgraded to gomplate 4: https://github.com/hairyhenderson/gomplate/issues/1872)
```

I think committing the rest of the comments is a little excessive.

Reply:
Thanks @risicle, could you clarify what's the harm in leaving those comments in? (I added them because I thought they could help future me updating the package)

Reply:
:shrug: if everyone kept in the commented-out scraps of variations they could have included it would be unmanageable.

Not important though.

Reply:
I still don't follow: it's not "everyone" it's just one person - and not some random guy, I am the sole maintainer of this package.

Anyway, for future changes I'll try and keep a separate set of separate, private notes in order not to upset people. Thanks for merging this.